project-backup/workspace-recovery.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+set -euo pipefail
+set -x
+
+# --- Configuration ---
+: "${DEVWORKSPACE_BACKUP_REGISTRY:?Missing DEVWORKSPACE_BACKUP_REGISTRY}"
+: "${DEVWORKSPACE_NAMESPACE:?Missing DEVWORKSPACE_NAMESPACE}"
+: "${DEVWORKSPACE_NAME:?Missing DEVWORKSPACE_NAME}"
+: "${BACKUP_SOURCE_PATH:?Missing BACKUP_SOURCE_PATH}"
+
+BACKUP_IMAGE="${DEVWORKSPACE_BACKUP_REGISTRY}/backup-${DEVWORKSPACE_NAMESPACE}-${DEVWORKSPACE_NAME}:latest"
+
+# --- Functions ---
+backup() {
+  TARBALL_NAME="devworkspace-backup.tar.gz"
+  cd /tmp
+  echo "Backing up devworkspace '$DEVWORKSPACE_NAME' in namespace '$DEVWORKSPACE_NAMESPACE' to image '$BACKUP_IMAGE'"
+
+  # Create tarball of the backup source path
+  tar -czvf "$TARBALL_NAME" -C "$BACKUP_SOURCE_PATH" .
+
+  # Push the tarball to the OCI registry using oras as a custom artifact
+  oras_args=(
+    push
+    "$BACKUP_IMAGE"
+    --artifact-type application/vnd.devworkspace.backup.artifact.v1+json
+    --annotation devworkspace.name="$DEVWORKSPACE_NAME"
+    --annotation devworkspace.namespace="$DEVWORKSPACE_NAMESPACE"
+    --disable-path-validation
+  )
+  if [[ -n "${REGISTRY_AUTH_FILE:-}" ]]; then
+    oras_args+=(--registry-config "$REGISTRY_AUTH_FILE")
+  fi
+  if [[ -n "${ORAS_EXTRA_ARGS:-}" ]]; then
+    extra_args=( ${ORAS_EXTRA_ARGS} )
+    oras_args+=("${extra_args[@]}")
+  fi
+  oras_args+=("$TARBALL_NAME")
+  oras "${oras_args[@]}"
+  rm -f "$TARBALL_NAME"
+
+  echo "Backup completed successfully."
+}
+
+restore() {
+  local container_name="workspace-restore"
+
+  podman create --name "$container_name" "$BACKUP_IMAGE"
```

Comment:
You [mentioned](https://github.com/devfile/devworkspace-operator/pull/1530#issuecomment-3575133867) that you had replaced `podman` with `oras` for backup creation. Also, @rohanKanojia [noticed](https://github.com/devfile/devworkspace-operator/pull/1530#issuecomment-3576004380) that a different format of the backup image was used. 
I think, you need to refactor this function as well to switch to `oras` to restore the ability to restore backups.